pkg/client/src/app/common/KeycloakProvider.tsx

```diff
@@ -14,6 +15,17 @@ interface IKeycloakProviderProps {
 export const KeycloakProvider: React.FunctionComponent<
   IKeycloakProviderProps
 > = ({ children }) => {
+  const checkAuthCookie = () => {
+    let currentCookie = getCookie("keycloak_cookie");
+    if (currentCookie !== "" && currentCookie !== null) {
+    } else {
+      currentCookie = keycloak?.token || "";
+
+      if (currentCookie != "" && currentCookie != null) {
```

Comment:
Nit: can we avoid `==` and `!=` and replace them with `===` and `!==`? It doesn't particularly matter here but it's good to be consistent.

Also, this line could also become `if (currentCookie) {`. It feels a little weird to nest them though right?

It seems like this whole block of code is: `Check whether the cookie is set, and if not, set it to the keycloak token if present`. Since `currentCookie` is never used outside this function anyway, I think you could actually just simplify the whole function to this:

```ts
  const checkAuthCookie = () => {
    if (!getCookie("keycloak_cookie") && keycloak?.token) {
      setCookie("keycloak_cookie", keycloak.token, 365);
    }
  };
```